src/app/helptext/data-protection/replication/replication-wizard.ts

```diff
@@ -48,6 +48,8 @@ export default {
  <code>/zvol1</code> after <i>dataset1</i> will create <i>zvol1</i> for \
  snapshot storage.'),
 
+  sudo_warning: T(`Selected SSH connection uses non-root user. Would you like to use passwordless sudo with <i>/usr/sbin/zfs</i> commands?
```

Comment:
We should say "Would you like to use sudo with <i>/usr/sbin/zfs</i> commands? Passwordless `sudo` must be enabled on the remote system" because this option does not enable passwordless sudo on the remote system (like it might seem from this dialog test